src/utilities/include/vec.h

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 #pragma once
+#if MANIFOLD_EXCEPTIONS
 #include <exception>
+#endif
```

Comment:
Do we need to include this per file, or can it just be wherever we define `ASSERT`?

Reply:
yeah probably just need it when we define `ASSERT`